src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
Linking to a SCSS file directly is not valid. SCSS needs to be compiled to CSS before being used in an HTML file. Please compile 'styles/main.scss' to 'styles/main.css' and update the href attribute to point to the CSS file.